src/assets/css/videoosd.css

```diff
@@ -248,8 +248,8 @@
 }
 
 @media all and (max-width: 30em) {
-    .btnFastForward,
-    .btnRewind,
+    .osdControls .btnFastForward,
+    .osdControls .btnRewind,
```

Comment:
These lines are probably not required because of this https://github.com/jellyfin/jellyfin-web/blob/bd9250f946890c7c4fcbc6669849a5d83a587051/src/assets/css/videoosd.css#L272-L277
But no harm.